src/peripherals.rs

```diff
@@ -10,6 +10,7 @@ use crate::mac;
 use crate::modem;
 #[cfg(any(esp32, esp32s2, esp32s3, esp32c6))]
 use crate::pcnt;
+#[cfg(feature = "legacy_rmt")]
```

Comment:
if you are introducing this feature, you also need to first define it inside the `Cargo.toml` similar how it was done for the adc-oneshot-legacy.

Reply:
Done

Reply:
Done